src/leases/components/leaseSections/summary/LeaseHistory.js

```diff
@@ -0,0 +1,197 @@
+// @flow
+import React, {PureComponent} from 'react';
+import {connect} from 'react-redux';
+
+import LeaseHistoryItem from './LeaseHistoryItem';
+import TitleH3 from '$components/content/TitleH3';
+import {LeaseFieldPaths, LeaseFieldTitles} from '$src/leases/enums';
+import {getContentRelatedLeasesFrom, getContentRelatedLeasesTo, sortRelatedLeasesFrom} from '$src/leases/helpers';
+import {getFieldOptions} from '$src/util/helpers';
+import {getUiDataLeaseKey} from '$src/uiData/helpers';
+import {
+  getAttributes as getLeaseAttributes,
+  getCurrentLease,
+} from '$src/leases/selectors';
+
+import type {Attributes} from '$src/types';
+import type {Lease} from '$src/leases/types';
+import { restructureLease, sortRelatedHistoryItems } from "$src/leases/helpers";
+
+type Props = {
+  currentLease: Lease,
+  leaseAttributes: Attributes,
+}
+
+type State = {
+  currentLease: Lease,
+  leaseAttributes: Attributes,
+  relatedLeasesFrom: Array<Object>,
```

Comment:
Is it possible to use something like Array<Lease> here? Not familiar with this typing system